app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt

```diff
@@ -216,23 +230,22 @@ class SettingsFragment constructor(
                     }
                 }
             }
-            findPreference<SwitchPreference>("crash_reporting")?.let {
-                it.isVisible = true
-                it.setOnPreferenceChangeListener { _, newValue ->
-                    val checked = newValue as Boolean
-
-                    true
-                }
+        }
+        findPreference<SwitchPreference>("crash_reporting")?.let {
+            it.isVisible = BuildConfig.FLAVOR == "full"
+            it.setOnPreferenceChangeListener { _, newValue ->
+                val checked = newValue as Boolean
+                true
             }
+        }
 
-            val pm = requireContext().packageManager
-            val hasWearApp = pm.getLaunchIntentForPackage("com.google.android.wearable.app")
-            val hasSamsungApp = pm.getLaunchIntentForPackage("com.samsung.android.app.watchmanager")
-            findPreference<PreferenceCategory>("wear_category")?.isVisible = hasWearApp != null || hasSamsungApp != null
-            findPreference<Preference>("wear_settings")?.setOnPreferenceClickListener {
-                startActivity(SettingsWearActivity.newInstance(requireContext()))
-                return@setOnPreferenceClickListener true
-            }
+        val pm = requireContext().packageManager
+        val hasWearApp = pm.getLaunchIntentForPackage("com.google.android.wearable.app")
+        val hasSamsungApp = pm.getLaunchIntentForPackage("com.samsung.android.app.watchmanager")
+        findPreference<PreferenceCategory>("wear_category")?.isVisible = hasWearApp != null || hasSamsungApp != null
```

Comment:
We should probably add a `full` check here for the Wear OS category to be hidden from minimal like before

Reply:
Fixed